classes/class.pmproemail.php

```diff
@@ -264,13 +297,12 @@ function sendCheckoutEmail($user = NULL, $invoice = NULL)
 			$this->data = array(
 								"subject" => $this->subject, 
 								"name" => $user->display_name, 
-								"display_name" => $user->display_name,
```

Comment:
We should probably keep this email template variable in case live sites are already using it.

Reply:
Sure thing, but it's duplicated. The display_name key is already at the end of that array of data.

Reply:
Merged from dev resolving this redundant key https://github.com/strangerstudios/paid-memberships-pro/pull/2060/commits/c54b9ebef6fe0934cacb59156f52c9c848c6bd21